docs/4.x/tests-in-depth-faq.md

```diff
@@ -0,0 +1,144 @@
+---
+category: DevelopInDepth
+---
+# Tests FAQ 
+
+## How do I modify the config ini to test certain conditions?
+
+### Integration and system PHP tests
+
+You can use the [configuration object](/guides/piwiks-ini-configuration) as usual and set any specific ini setting before executing the test.
+
+You don't need to reset the ini after each test as this is done automatically for you.
```

Comment:
not sure if this is also true for system tests?

Reply:
not true for system tests, just integration

Reply:
👍  thanks for this